global.json

```diff
@@ -3,6 +3,6 @@
         "src"
     ],
     "sdk": {
-        "version": "2.1.603"
+        "version": "2.1.700"
```

Comment:
How about 2.1.701 ?
https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.12/2.1.12.md

Reply:
Oh sure, it was released yesterday ;)